src/test/java/core/basesyntax/dataio/FileReaderImplTest.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.dataio;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    @Test
+    public void inputOk() {
+        List<String> expected = new ArrayList<>();
+        expected.add("b,banana,15");
+        expected.add("b,apple,25");
+        expected.add("s,banana,60");
+        FileReader fileReader = new FileReaderImpl("src/main/resources/inputdata_test.csv");
```

Comment:
```suggestion
        ... = new FileReaderImpl().read("src/test/resources/inputdata_test.csv");
```
resources that are used for tests should be located in `src/test` folder